src/hdtSkinnedMesh/hdtSkinnedMeshAlgorithm.cpp

```
@@ -605,57 +564,68 @@ namespace hdt
+			newPt.m_positionWorldOnB = worldB;
+			newPt.m_combinedFriction = rb0->m_rig.getFriction() * rb1->m_rig.getFriction();
+			newPt.m_combinedRestitution = rb0->m_rig.getRestitution() * rb1->m_rig.getRestitution();
+			newPt.m_combinedRollingFriction = rb0->m_rig.getRollingFriction() * rb1->m_rig.getRollingFriction();
+			maniford->addManifoldPoint(newPt);
 		}
 	}
 
 	template <class T0, class T1>
 	void SkinnedMeshAlgorithm::processCollision(T0* shape0, T1* shape1, MergeBuffer& merge, CollisionResult* collision)
 	{
 		int count = std::min(checkCollide(shape0, shape1, collision), MaxCollisionCount);
```

Clang (14.0.6)

[warning] variable 'count' is not initialized

(cppcoreguidelines-init-variables)